apps/fz_http/lib/fz_http_web/controllers/json/fallback_controller.ex

```diff
@@ -6,21 +6,6 @@ defmodule FzHttpWeb.JSON.FallbackController do
   """
   use FzHttpWeb, :controller
 
-  # This clause is an example of how to handle resources that cannot be found.
-  def call(conn, {:error, :not_found}) do
```

Comment:
This is used in new tests :)

Reply:
K. Restored it. I noticed the `assert_error_sent 404, fn -> ...` weren't hitting this code path (strangely?) so I had removed it.